internal/pkg/dl/agent.go

```diff
@@ -67,28 +66,3 @@ func FindAgent(ctx context.Context, bulker bulk.Bulk, tmpl *dsl.Tmpl, name strin
 
 	return agent, nil
 }
-
-func FindOfflineAgents(ctx context.Context, bulker bulk.Bulk, policyID string, unenrollTimeout time.Duration, opt ...Option) ([]model.Agent, error) {
-	o := newOption(FleetAgents, opt...)
-	past := time.Now().UTC().Add(-unenrollTimeout).Format(time.RFC3339)
-	res, err := Search(ctx, bulker, QueryOfflineAgentsByPolicyID, o.indexName, map[string]interface{}{
```

Comment:
Does anything else use the `prepareOfflineAgentsByPolicyID` var/query?

Reply:
Good spot, I've removed it 👍